src/components/trashlookup.jsx

```diff
@@ -1,156 +1,157 @@
-  // }
-  handleAddressSelect = address => {
-    // console.log("address:" + address.address1);
-    this.setState({
-      selectedAddress: address.address1,
-      searchQuery: address.address1,
-      issearch: 3
-    });
-  };
-  handleSearch = query => {
-    console.log("handel Search:" + query);
-    if (query.length === 0) {
-      this.setState({ searchQuery: query, issearch: 0 });
-    } else {
-      this.setState({ searchQuery: query, issearch: 1 });
-    }
-  };
-
-  handleSearchClick = query => {
-    if (this.state.searchQuery.length === 0) {
-      alert("search criteria misssing");
-    } else {
-      this.setState({ issearch: 2 });
-    }
-    console.log("button is click");
-  };
-
-  getSearchResult = () => {
-    const searchQuery = this.state.searchQuery;
-    const issearch = this.state.issearch;
-    let filtered = ""; // PostData;
-    // console.log("value of searchQuery " + searchQuery);
-    //console.log("value of isearch is " + issearch);
-    if (issearch === 1) {
-      console.log("searchclicked - false ");
-      if (searchQuery)
-        filtered = PostData.filter(m =>
-          m.address1.toLowerCase().startsWith(searchQuery.toLowerCase())
-        );
-    } else {
-      if (issearch === 2) console.log("searchclicked - true ");
-      filtered = PostData.filter(m =>
-        m.address1.toLowerCase().startsWith(searchQuery.toLowerCase())
-      );
-    }
-
-    console.log(filtered.length);
-    console.log(filtered);
-    return { totalCount: filtered.length, data: filtered, issearch: issearch };
-  };
+  recylceNextCollectionDate() {
+    const dayOfWeek = moment().day();
+    return dayOfWeek >= 0 && dayOfWeek < 5
+      ? moment()
+          .add(5 - dayOfWeek, "d")
+          .format("D/M/YYYY")
+      : moment()
+          .add(6, "d")
+          .format("D/M/YYYY");
+  }
+
+  trashNextCollectionDate() {
+    const dayOfWeek = moment().day();
+
+    return dayOfWeek >= 1 && dayOfWeek < 6
+      ? moment()
+          .add(6 - dayOfWeek, "d")
+          .format("D/M/YYYY")
+      : dayOfWeek === 6
+      ? moment()
+          .add(2, "d")
+          .format("D/M/YYYY")
+      : moment()
+          .add(1, "d")
+          .format("D/M/YYYY");
+  }
 
   render() {
-    const { totalCount, data: data, issearch } = this.getSearchResult();
+    const {  data } = this.getAddresses();
+    const isHidden = this.state.isHidden;
+    const firstText = " First paragraph shown here ";
+    const secondText = "  second paragraph shown here ";
 
-    const searchQuery = this.state.addresses;
-    const searchValue = this.state.searchQuery;
     return (
       <React.Fragment>
-        <p>Showing {totalCount} address in the database.</p>
-        <SearchBox
-          value={searchValue}
-          onChange={this.handleSearch}
-          onClick={this.handleSearchClick}
-        />
+  
+        <h6>Find Your Collection Schedule.</h6>
+        <p>{firstText}</p>
```

Comment:
there is no benefit of storing first text  and second text in a variable, you can just go ahead and put that in the jsx below.